Src/BootCamp.Chapter/ConsoleLogger.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class ConsoleLogger : ILogger
+    {
+        public void Error(Exception ex)
+        {
+            Console.WriteLine($"{System.DateTime.Now} - Exception has been thrown: {ex} {Environment.NewLine}");
+        }
+
+        public void Error(Exception ex, string message)
+        {
+            Error(ex);
+            Console.WriteLine(message);
+        }
+
+        public void LogExit()
```

Comment:
Logger should just log. Now what contents the logging message will have should come as input. The only requirement is that it has a  "date - " at the start. So please make the change.